helm_deploy/licences/templates/_envs.tpl

```diff
@@ -153,4 +153,10 @@ env:
 
   - name: REDIS_TLS_ENABLED
     value: "true"
+
+  - name: HDC_URL
+    value: "http://licences:80"
+
+  - name: GOTENBERG_API_URL
+    value: "http://licences-gotenberg:3000"
```

Comment:
The URL prefix the licences app to use when POSTing HTML documents to the licences-gotenberg service for conversion to PDF.